addon/components/nrg-appbar.js

```diff
@@ -17,7 +17,7 @@ export default class NrgAppbarComponent extends Component {
     const productionEnvironments = this.application.environmentConfig?.[
       'ember-nrg-ui'
     ]?.productionEnvironments ?? ['prod'];
-    const environment = this.application.applicationSettings.localEnvironment;
+    const environment = this.application.settings?.localEnvironment;
```

Comment:
Can't we guarantee that `this.application.settings` will exist?

Reply:
Probably, though I don't think it hurts to account for the possibility that it doesn't.